### PR TITLE
fix: bn_stats may be undefined when batchnorm_file is empty

### PR DESCRIPTION
**Pull request by andrewwhitecdw**
### Problem
fix: bn_stats may be undefined when batchnorm_file is empty

### Fix
Replace:
```
    n_bn_updated = 0
    global_state_dict = net.state_dict()
    if batchnorm_file:
```

with:
```
    n_bn_updated = 0
    bn_stats = None
    global_state_dict = net.state_dict()
    if batchnorm_file:
```

### Files changed
- `FLGradientInversion/main.py`

### CODE REVIEW

**Comment by andrewwhitecdw** [comment]
Closing this sweep-generated PR: sole commit is missing a valid Signed-off-by trailer. It does not meet the sweep requirements (single signed-off commit).
